pkg/cmd/cli/secrets/new.go

```diff
@@ -47,12 +64,10 @@ func NewCmdCreateSecret(name, fullName string, f *clientcmd.Factory, out io.Writ
 	options.Out = out
 
 	cmd := &cobra.Command{
-		Use:   fmt.Sprintf("%s NAME SOURCE [SOURCE ...]", name),
-		Short: "Create a new secret based on a file or files within a directory",
-		Long: fmt.Sprintf(`Create a new secret based on a file or files within a directory.
-
-  $ %s <secret-name> <source> [<source>...]
-		`, fullName),
+		Use:     fmt.Sprintf("%s NAME SOURCENAME=SOURCE [SOURCENAME=SOURCE ...]", name),
```

Comment:
@smarterclayton how to represent this? `[SOURCENAME]=SOURCE [[SOURCENAME]=SOUCE...]`